### ISSUE TITLE
Emojis In the Titles and config file.

### ISSUE THREAD

**Issue by Setherizor** [hivickylai/hugo-theme-introduction#43]
Hello! I am trying to use some emojis in the tagLine of this theme. Is there something I am missing? How would we go about using real emojis :smile: with this theme. Any help is much appreciated!

It would be amazing if this
```
[params]
    tagLine = "Code Wizard :computer: - Student :pencil2: - Friend :laughing:"
```
Would render into something like this
## Computer Lover :computer: - Student :pencil2: - Friend :laughing:

**Comment by pmviva** [comment]
I guess we need to call Hugo's emojify function in the templates when we print the tagline.

In file layouts/index.html you need to have:
```
<h3 class="subtitle is-3 {{ if .Site.Params.fadeIn | default true }}fade-in two{{ end }}">
  {{ .Site.Params.tagLine | emojify }}
</h3>
```
Instead of:
```
<h3 class="subtitle is-3 {{ if .Site.Params.fadeIn | default true }}fade-in two{{ end }}">
  {{ .Site.Params.tagLine }}
</h3>
```
You can submit a pull request with the change :smiley:

**Comment by Setherizor** [comment]
Thank you! I'll submit the pull soon.

**Comment by hivickylai** [comment]
Addressed in #44
